lib/trino-filesystem-s3/src/main/java/io/trino/filesystem/s3/S3FileSystemConfig.java

```diff
@@ -184,6 +210,19 @@ public S3FileSystemConfig setStsRegion(String stsRegion)
         return this;
     }
 
+    @NotNull
+    public ObjectCannedAcl getCannedAcl()
+    {
+        return objectCannedAcl;
+    }
+
+    @Config("s3.canned-acl")
```

Comment:
Pls add `@ConfigDescription`.
The configuration key by itself doesn't say very much.